internal/cmd/local/docker/docker.go

```diff
@@ -153,3 +175,205 @@ func (d *Docker) Port(ctx context.Context, container string) (int, error) {
+	// docker run
+	// -e POSTGRES_USER=docker -e POSTGRES_PASSWORD=docker -e POSTGRES_DB=airbyte -e PGDATA=/var/lib/postgresql/data \
+	// -v ~/.airbyte/abctl/data/airbyte-volume-db/pgdata:/var/lib/postgresql/data
+	// postgres:13-alpine
+	conTransform, err := d.Client.ContainerCreate(
+		ctx,
+		&container.Config{
+			Image: "postgres:13-alpine",
+			Env: []string{
+				"POSTGRES_USER=docker",
+				"POSTGRES_PASSWORD=docker",
+				"POSTGRES_DB=postgres",
+				"PGDATA=" + migratePGDATA,
+			},
+		},
+		&container.HostConfig{
+			Mounts: []mount.Mount{{
+				Type:   mount.TypeBind,
+				Source: dst,
+				Target: migratePGDATA,
+			}},
+		},
+		nil,
+		nil,
+		"")
+	if err != nil {
+		return fmt.Errorf("could not create docker container: %w", err)
+	}
+	pterm.Debug.Println(fmt.Sprintf("Created secondary migration container '%s'", conTransform.ID))
+	pterm.Debug.Println(fmt.Sprintf("Container was created with the following warnings: %s", conTransform.Warnings))
+	if err := d.Client.ContainerStart(ctx, conTransform.ID, container.StartOptions{}); err != nil {
+		return fmt.Errorf("could not start container %s: %w", conTransform.ID, err)
+	}
+	// cleanup and remove container when we're done
+	defer func() { d.stopContainer(ctx, conTransform.ID) }()
+
+	// TODO figure out a better way to determine when the container has successfully started
+	time.Sleep(10 * time.Second)
+
+	// docker exec airbyte-abctl-migrate psql -U docker -c "CREATE ROLE airbyte SUPERUSER CREATEROLE CREATEDB REPLICATION BYPASSRLS LOGIN PASSWORD 'airbyte'"
+	// docker exec airbyte-abctl-migrate psql -U airbyte postgres -c 'ALTER DATABASE airbyte RENAME TO "db-airbyte"'
+	var (
+		cmdPsqlRename = []string{"psql", "-U", "docker", "-d", "postgres", "-c", `ALTER DATABASE "airbyte" RENAME TO "db-airbyte"`}
+		cmdPsqlUser   = []string{"psql", "-U", "docker", "-d", "postgres", "-c", `CREATE ROLE airbyte SUPERUSER CREATEROLE CREATEDB REPLICATION BYPASSRLS LOGIN PASSWORD 'airbyte'`}
+	)
+	// add a new database user to match the default helm user
+	now := time.Now()
+	pterm.Debug.Println("Adding Airbyte postgres user")
+	if err := d.exec(ctx, conTransform.ID, cmdPsqlUser); err != nil {
+		pterm.Debug.Println("Failed to add postgres user")
+		return fmt.Errorf("could not update postgres user: %w", err)
+	}
+	pterm.Debug.Println(fmt.Sprintf("Adding Airbyte postgres user completed in %s", time.Since(now)))
+
+	// rename the database to match the default helm database name
+	pterm.Debug.Println("Renaming database")
+	now = time.Now()
+	if err := d.exec(ctx, conTransform.ID, cmdPsqlRename); err != nil {
+		pterm.Debug.Println("Failed to rename database")
+		return fmt.Errorf("could not rename postgres database: %w", err)
+	}
+	pterm.Debug.Println(fmt.Sprintf("Renaming database completed in %s", time.Since(now)))
+
+	return nil
+}
+
+// volumeExists returns true if the volumeID exists on the host machine, false otherwise.
+func (d *Docker) volumeExists(ctx context.Context, volumeID string) string {
```

Comment:
1. The comment here is out of date
2. I think it might be better if this returned a `(bool, string)` where IFF `true` then the send value is the mountpoint

Reply:
I'll update the comment, but will leave the signature the same as the boolean return value would always be linked to the string value. Either it would be `true` and a non-empty string, or `false` and an empty string, rendering the boolean return value redundant.